### ISSUE TITLE
Item box loses focus when tabbing out of field with button after deleting contents

### ISSUE THREAD

**Issue by AbeJellinek** [zotero/zotero#4241]
e.g. DOI. Select all the text in the field, delete, press Tab. Focus moves to View Online and then is lost when that button hides.

https://forums.zotero.org/discussion/comment/465664/#Comment_465664

**Comment by abaevbog** [comment]
I added a tweak to address this in https://github.com/zotero/zotero/pull/4178 as it has a few other focus-related fixes.

I ran into it before I saw the comment on the forums and I should've created an issue for it then.

**Comment by dstillman** [comment]
https://forums.zotero.org/discussion/comment/466290/#Comment_466290

I can still reproduce the original issue above.

**Comment by abaevbog** [comment]
Oh, I see. Apologies, I did address only part of the general issue from the forums thread with lost focus (one with focus being lost on tab after any kind of editing that is not deletion). That was due to lack of `id`s on buttons, so they could never be refocused at all.

If the content is emptied, though, the actual options button is never rendered, so there is no node to refocus (regardless of if it is mean to have an `id` or not). I'll tweak it to maybe look for the editable field from that row and simulate a tab from that to focus the next row
